src/server/components/lists/questionnaire.ts

```diff
@@ -42,8 +46,8 @@ export const questions: Questions = {
       return `In which country do you need a ${serviceLabel}?`;
     },
     needsToAnswer(req: Request) {
-      const { country } = getAllRequestParams(req);
-      return country === undefined || country === "";
+      const { sameCountry, country } = getAllRequestParams(req);
+      return ((sameCountry === "yes" && stringNotSpecified(country)) || ((stringNotSpecified(sameCountry) || sameCountry === "no") && stringNotSpecified(country)));
```

Comment:
soz, what's this code trying to do? 

if they answered "yes" to sameCountry, they don't have to answer country field,
if they answered "no" to sameCountry, they have to answer the country field?


```suggestion
      const { sameCountry, country } = getAllRequestParams(req);
      
      if(sameCountry === "yes")  {
        return false
      }
      
      const countryFieldUnanswered = !!country; // or stringNotSpecified(..)  !! is only unsafe when number is involved
      // sameCountry must either be "no", empty, or invalid input, all of which have the same result, so no need to deal with it here?
      
      return countryFieldUnanswered;
      
```